docs/syntax_and_semantics/macros/README.md

```diff
@@ -291,6 +291,21 @@ Foo.new.describe # => "Class is Foo"
 Foo.describe     # => "Class is Foo"
 ```
 
+## The top level module
+
+It is posible to access the top level module, as a [`TypeNode`](https://crystal-lang.org/api/latest/Crystal/Macros/TypeNode.html), with a special instance variable: `@top_level`. The following example shows its utility:
```

Comment:
```suggestion
It is possible to access the top-level namespace, as a [`TypeNode`](https://crystal-lang.org/api/latest/Crystal/Macros/TypeNode.html), with a special variable: `@top_level`. The following example shows its utility:
```

The first two changes are just typos. The other two are semantic nitpicking, feel free to ignore.
I'm uncomfortable with calling the top-level a module. "namespace" seems more appropriate to me, to avoid possible misinterpretation.
And while `@top_level` uses the syntax of an ivar, it is technically a global variable (or rather even a constant). I wouldn't call it *instance* variable.